README.md

```diff
@@ -1 +0,0 @@
-# django-girls-blog-RootenberG
```

Comment:
unrelated

Reply:
I deleted this file.

Reply:
You added a commit with the file deletion. I want this change to not appear in this PR.

Reply:
Besides, it now creates a merge conflict and you'd not be able to merge if even with approvals in place.